Assets/MRTK/Core/Interfaces/SpatialAwareness/Observers/Experimental/IMixedRealityOnDemandObserver.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.SpatialAwareness
+{
+    /// <summary>
+    /// The interface for defining an <see cref="IMixedRealityOnDemandObserver"/> which enables on demand updating
```

Comment:
```suggestion
    /// The interface for defining an on-demand spatial observer which enables on demand updating
```

Otherwise, you are creating a doc loop :)